values.yaml

```diff
@@ -66,6 +66,7 @@ config:
   analytics:
     token: "0c39f8410fd548bfa1976957d0248289"
   tranql_enabled: "true"
+  hidden_support_section: []
```

Comment:
this might lead to some unexpected behavior if what's being passed in isn't a string. What would be some example values for this field?

Reply:
That's true, I thought about that as well, so the example values for this field would be 'community' or 'documentation', which stands for corresponding components on the support page. UI will only hide the section if each value matches the section name (i.e. UI will only hide the community section if 'community' is in this array). So if there are irrelevant values (i.e. ['ui', 'support'] ), UI will just ignore it. I should probably put some example values here instead of in the ui repo. How do you think of this approach?

Reply:
I think if that's the case, this value here should be a comma-separated string, and it should be turned into an array in the env parsing script